regression-test/data/jdbc_catalog_p0/test_mysql_jdbc_catalog.out

```diff
@@ -297,7 +297,7 @@ sys
 
 -- !mysql_all_types --
 \N	302	\N	502	602	4.14159	\N	6.14159	\N	-124	-302	2013	-402	-502	-602	\N	2012-10-26T02:08:39.345700	2013-10-26T08:09:18	-5.14145	\N	-7.1400	row2	\N	09:11:09.567	text2	0xe86f6c6c6f20576f726c67	\N	\N	0x2f	\N	0x88656c6c9f	Value3
-201	301	401	501	601	3.14159	4.1415926	5.14159	true	-123	-301	2012	-401	-501	-601	2012-10-30	2012-10-25T12:05:36.345700	2012-10-25T08:08:08	-4.14145	-5.1400000001	-6.1400	row1	line1	09:09:09.567	text1	0x48656c6c6f20576f726c64	{"age":30,"city":"London","name":"Alice"}	Option1,Option3	0x2a	0x48656c6c6f00000000000000	0x48656c6c6f	Value2
-202	302	402	502	602	4.14159	5.1415926	6.14159	false	-124	-302	2013	-402	-502	-602	2012-11-01	2012-10-26T02:08:39.345700	2013-10-26T08:09:18	-5.14145	-6.1400000001	-7.1400	row2	line2	09:11:09.567	text2	0xe86f6c6c6f20576f726c67	{"age":18,"city":"ChongQing","name":"Gaoxin"}	Option1,Option2	0x2f	0x58676c6c6f00000000000000	0x88656c6c9f	Value3
-203	303	403	503	603	7.14159	8.1415926	9.14159	false	\N	-402	2017	-602	-902	-1102	2012-11-02	2023-07-10T09:34:21.204900	2013-10-27T08:11:18	-5.14145	-6.1400000000001	-7.1400	row3	line3	09:11:09.567	text3	0xe86f6c6c6f20576f726c67	{"age":24,"city":"ChongQing","name":"ChenQi"}	Option2	0x2f	0x58676c6c6f00000000000000	\N	Value1
+201	301	401	501	601	3.14159	4.1415926	5.14159	1	-123	-301	2012	-401	-501	-601	2012-10-30	2012-10-25T12:05:36.345700	2012-10-25T08:08:08	-4.14145	-5.1400000001	-6.1400	row1	line1	09:09:09.567	text1	0x48656c6c6f20576f726c64	{"age":30,"city":"London","name":"Alice"}	Option1,Option3	0x2a	0x48656c6c6f00000000000000	0x48656c6c6f	Value2
+202	302	402	502	602	4.14159	5.1415926	6.14159	0	-124	-302	2013	-402	-502	-602	2012-11-01	2012-10-26T02:08:39.345700	2013-10-26T08:09:18	-5.14145	-6.1400000001	-7.1400	row2	line2	09:11:09.567	text2	0xe86f6c6c6f20576f726c67	{"age":18,"city":"ChongQing","name":"Gaoxin"}	Option1,Option2	0x2f	0x58676c6c6f00000000000000	0x88656c6c9f	Value3
+203	303	403	503	603	7.14159	8.1415926	9.14159	0	\N	-402	2017	-602	-902	-1102	2012-11-02	2023-07-21T17:30:10.974300	2013-10-27T08:11:18	-5.14145	-6.1400000000001	-7.1400	row3	line3	09:11:09.567	text3	0xe86f6c6c6f20576f726c67	{"age":24,"city":"ChongQing","name":"ChenQi"}	Option2	0x2f	0x58676c6c6f00000000000000	\N	Value1
```

Comment:
Why does the value of the timestamp changed here？

Reply:
done